airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -371,7 +371,12 @@ def test_parse_response(self, requests_mock, stream_cls):
 
         requests_mock.get(STREAM_URL, json={response_field: expected})
         test_response = requests.get(STREAM_URL)
-        output = list(stream.parse_response(test_response, None))
+
+        if isinstance(stream, TicketMetrics):
```

Comment:
can we split this in a separate test dedicated to ticket metrics?

Reply:
Good call. Split for clarity.